pkg/controller/certificaterequests/sync_test.go

```diff
@@ -139,7 +139,7 @@ func TestSync(t *testing.T) {
 		gen.SetCertificateRequestCSR(csr),
 		gen.SetCertificateRequestIssuer(cmapi.ObjectReference{
 			Kind: "Issuer",
-			Name: "fake-issuer",
+			Name: "ca-issuer",
```

Comment:
AH, *this* might explain why you've changed the name to `ca-issuer` everywhere? Presumably we were faking out the IssuerFactory in the past, which meant it didn't matter that `fake-issuer` didn't equal the old name (`test`) before?

Reply:
Sure, can revert these to make it more clear